hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/operations/SplitBrainMergeOperation.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
```

Comment:
Copyright needs to be adjusted to 2018 after rebase, sorry (same for all other new files).